### DISCUSSION TITLE
Rounded corner for window

### DISCUSSION THREAD

**Discussion by Dmitriy147** [fyne-io/fyne#4338] [General]
Is it any way to create window (modale window) with rounded corner?
And may be anybody could show some example with rounded corner for button?

**Comment by Jacalz** [comment]
Om what operating system? Windows 11 and the latest macOS versions round the bottom corners automagically I think.

**Comment by Dmitriy147** [comment]
I would like to modify the corner radius manualy..

**Comment by andydotxyz** [comment]
The OS controls the window corners so we can't override it.

For rounded button corners just use Fyne v2.4.0 or later (there is also v2.4.1) and they are rounded by default. The radius is controlled by the theme through the InputRadius size.

**Comment by Dmitriy147** [comment]
Thanks a lot!
